### PR TITLE
feat: 1006 Wire sell order action

### PR DESCRIPTION
**Pull request by flagrede**
## Description

Closes: regen-network/regen-registry#1006

Update `useCreateSellOrderSubmit.tsx` with the actual create sell order transaction.
Also made some changes to the confirmation modal and the redirection.

Now we have sell order data again on the [v4 testnet](http://194.37.81.19:1317/regen/ecocredit/marketplace/v1/sell-orders)  🎉 

---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...

- [x] provided a link to the relevant issue or specification
- [x] provided instructions on how to test
- [x] reviewed "Files changed" and left comments if necessary
- [x] confirmed all CI checks have passed
- [ ] once the PR is closed, set up backport PRs for `redwood` and `hambach` (see below)

#### Setting up backport PRs

After merging your PR to `master`, set up backports by doing the following:

1. If your branch does not have merge commits, add the following comment to
   your PR, `@Mergifyio backport redwood hambach`.

2. If your branch does have merge commits:

    a. Pull latest `master`, `hambach` and `redwood` branches

    b. Create new branches for backports and merge `master` (replace `<PR#>` with your PR #)
    ```
    git checkout -b hambach-backport-<PR#> hambach
    git merge master
    git push origin hambach-backport-<PR#>
    git checkout -b redwood-backport-<PR#> redwood
    git merge master
    git push origin redwood-backport-<PR#>`
    ```

    c. Open new PRs in regen-web targeting `hambach` and `redwood`, respectively.
  

### How to test

1. Go to https://deploy-preview-1052--regen-registry.netlify.app/ecocredits/dashboard
2. Create a sell order from action button on the right

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...

- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed code correctness and readability
- [ ] verified React components follow DRY principles
- [ ] reviewed documentation is accurate
- [ ] reviewed tests
- [ ] manually tested (if applicable)

### CODE REVIEW

File: web-registry/src/pages/MyEcocredits/MyEcocredits.tsx [hunk 1]
```diff
@@ -89,6 +93,13 @@ export const MyEcocredits = (): JSX.Element => {
     setError(undefined);
   };
 
+  const onButtonClick = (): void => {
+    handleTxModalClose();
+    if (txButtonTitle === CREATE_SELL_ORDER_BUTTON) {
```
Comment (flagrede) [inline web-registry/src/pages/MyEcocredits/MyEcocredits.tsx:98]: Redirect to the sell order table after sell order creation.

File: web-registry/src/pages/MyEcocredits/hooks/useUpdateTxModalTitle.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+type Props = {
+  deliverTxResponse?: DeliverTxResponse;
+  setTxModalTitle: useStateSetter<string | undefined>;
+};
+
+const SELL_ORDER_EVENT_TYPE = 'regen.ecocredit.marketplace.v1.EventSell';
+const SELL_ORDER_ATTRIBUTE_KEY = 'sell_order_id';
+
+export const useUpdateTxModalTitle = ({
```
Comment (flagrede) [inline web-registry/src/pages/MyEcocredits/hooks/useUpdateTxModalTitle.tsx:13]: This hook is used to update TxModalTitle based on information in the transaction response.
For the sell order we want to display `Sell Order #xxx`

**Comment by mhagel** [comment]
@flagrede This should be targeted to `v4` branch.

**Review by haveanicedavid — APPROVED** [review]
LGTM - nice work!

**Review by haveanicedavid — APPROVED** [review]

**Comment by flagrede** [comment]
@erikalogie You can now test this PR.
https://deploy-preview-1052--regen-registry.netlify.app/ecocredits/dashboard

**Comment by erikalogie** [comment]
@flagrede all of my ecocredits and basket tokens have been wiped. Is that something you can help me to get for testing? If so, I can send you my address.

**Comment by erikalogie** [comment]
@flagrede though now looking at David's PR for the create credit class form I have ecocredits on his deploy preview, just not at your deploy preview.

**Comment by flagrede** [comment]
@erikalogie Thanks for the feedbacks, will handle everything in this PR.

**Comment by flagrede** [comment]
@Mergifyio backport redwood hambach

**Pull request merged**